WORKSPACE

```diff
@@ -34,6 +34,10 @@ maven_install(
         # Gson used for decoding certificate status list
         "com.google.code.gson:gson:2.8.5",
 
+        "com.google.guava:guava:27.0.1-android",
```

Comment:
bazel.io seems to think we get ErrorProne for free: http://errorprone.info/docs/installation

Do we even need these deps?

Reply:
There's another similar fix here: https://github.com/protocolbuffers/protobuf/issues/5795 and even the bazel documentation references including the WORKSPACE and deps for it https://bazel.build/migrate/maven#guava-1. I added them because I was getting an error trying to run the test with `blaze test`.

Reply:
Weird, but ok!

Seems like the bazel docs are incorrect and/or missing some steps.